app/uk/gov/hmrc/cdsreimbursementclaim/models/claim/DeclarantType.scala

```diff
@@ -19,18 +19,18 @@ package uk.gov.hmrc.cdsreimbursementclaim.models.claim
 import julienrf.json.derived
 import play.api.libs.json.OFormat
 
-sealed trait DeclarantType extends Product with Serializable
+sealed trait DeclarantTypeAnswer extends Product with Serializable
 
-object DeclarantType {
-  case object Importer extends DeclarantType {
+object DeclarantTypeAnswer {
+  case object Importer extends DeclarantTypeAnswer {
     override def toString = "Importer"
```

Comment:
we need to remove this hardcoded values, right?

Reply:
Done